tests/Unit/IO/Observers/Test_Initialize.cpp

```diff
@@ -0,0 +1,61 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <cstddef>
+#include <unordered_map>
+#include <unordered_set>
+#include <utility>
+// IWYU pragma: no_include <exception>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "IO/Observer/ArrayComponentId.hpp"  // IWYU pragma: keep
+#include "IO/Observer/Initialize.hpp"
+#include "IO/Observer/Tags.hpp"  // IWYU pragma: keep
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+#include "tests/Unit/ActionTesting.hpp"
+
+namespace {
+template <typename Metavariables>
+struct observer_component
+    : ActionTesting::MockArrayComponent<Metavariables, size_t, tmpl::list<>,
+                                        tmpl::list<>> {
+  using simple_tags = typename observers::Actions::Initialize::simple_tags;
+  using initial_databox = db::compute_databox_type<simple_tags>;
```

Comment:
I think they are the same in this case, but why `simple_tags` instead of `return_tag_list` (i.e if  `compute_tags` were not empty, wouldn't I want `return_tag_list` ?)

Reply:
Because this is a component, not an action :) Currently we use `return_tag_list` inside the `Initialize` actions and `simple_tags`, `compute_tags`, and `initial_databox` in the components.

Reply:
sorry, I meant why not `using initial_databox = db::compute_databox_type<typename observers::Actions::Initialize::return_tag_list>` like on line 25 of ObserverComponent.hpp

Reply:
We use return_tag_list in DgElementArray, and I'm pretty sure it wouldn't work otherwise.

Reply:
Ah, this was to be consistent with what I did for the Actions that don't have a `return_tags_list`. I don't care which one we do.

Reply:
Okay, done.